tests/server/test_server.py

```diff
@@ -297,15 +297,6 @@ def test_version(client: FlaskClient, endpoint: str) -> None:
     assert response.data.decode("UTF-8") == expected
 
 
-@pytest.mark.parametrize("endpoint", ["/", "/stash", "/github"])
-def test_post_invalid_content_type(client: FlaskClient, endpoint: str) -> None:
```

Comment:
In recent flask versions, accessing `request.json` when the request has not the JSON mime-type automatically raises an error, so we do not need to handle this ourselves explicitly anymore (see last commit).